### PR TITLE
CFT to create a multi-AZ VPC

### PR DESCRIPTION
**Pull request by fabricetriboix**

### CODE REVIEW

File: CloudFormation/deploy-arkcase.yml [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+
+  InternetGatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId: !Ref Vpc
+      InternetGatewayId: !Ref InternetGateway
+
+  PublicSubnetA:
+    Type: AWS::EC2::Subnet
```
Comment (david-oc-miller) [inline CloudFormation/deploy-arkcase.yml:97]: @fabricetriboix, based on our task list, we said we wanted to specify a number of AZ's, although we didn't specify a subnet mix between AZ's ...  I guess each AZ would have one public, one private subnet.  How hard to extend this to support an arbitrary number of AZ's?

File: CloudFormation/deploy-arkcase.yml [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  PublicRouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref Vpc
+      Tags:
+        - Key: Env
+          Value: !Ref Env
```
Comment (david-oc-miller) [inline CloudFormation/deploy-arkcase.yml:250]: Can the route tables have names?  I see the Env and Project tags; but if all resources have name tags which include the VPC name, it's much easier to navigate in the console UI.

File: CloudFormation/deploy-arkcase.yml [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+Description: >
+  Deploy ArkCase
+
+Parameters:
+  Env:
+    Type: String
+    Description: Type of environment to provision
+    Default: prod
+    AllowedValues: [staging, prod]
```
Comment (david-oc-miller) [inline CloudFormation/deploy-arkcase.yml:13]: Let's consider more allowed values: dev, test, load-test.  I'm looking forward to when we live on AWS.

File: CloudFormation/deploy-arkcase.yml (hunk 1 above)
Comment (fabricetriboix) [reply CloudFormation/deploy-arkcase.yml:97]: Right now, that's two AZ with one public and one private subnet in each. Supporting an arbitrary number of AZs is a bit tricky, in the sense that CloudFormation doesn't support loops. It does support conditions, though, so it is possible to work around this limitation. I can go into more details if required.

File: CloudFormation/deploy-arkcase.yml (hunk 3 above)
Comment (fabricetriboix) [reply CloudFormation/deploy-arkcase.yml:13]: Sure! Thanks

File: CloudFormation/deploy-arkcase.yml (hunk 2 above)
Comment (fabricetriboix) [reply CloudFormation/deploy-arkcase.yml:250]: Yes, I can add a "Name" tag

**Pull request merged**